compiler/control/OMROptions.cpp

```diff
@@ -2695,8 +2706,11 @@ OMR::Options::jitPreProcess()
             {
             if (_aggressivenessLevel != -1) // -1 means not set
                {
-               if (OMR::Options::isAnyVerboseOptionSet())
+               if (OMR::Options::isAnyVerboseOptionSet()) {
```

Comment:
Same comment as above regarding the code style.

Reply:
writelinelocked